README.md

```diff
@@ -140,13 +140,25 @@ In order to use it, you need to create the blacklist table in a migration:
 def change
   create_table :jwt_blacklist do |t|
     t.string :jti, null: false
+    t.datetime :revoked_at, default: Time.now
+    t.datetime :expiration_time
```

Comment:
Following the convention of the `:jti` column name (which is the [JWT specification claim name for JWT ID](https://tools.ietf.org/html/rfc7519#section-4.1.7), I would call that column `exp`.